CTLD.lua

```diff
@@ -574,12 +734,12 @@ ctld.spawnableCrates = {
     },
     ["AA long range"] = {
         -- Patriot System
-        { weight = 555, desc = "Patriot Launcher", unit = "Patriot ln", side = 2 },
+        { weight = 555, desc = "Patriot Launcher", unit = "Patriot ln", side = 2, cratesRequired = 2},
```

Comment:
Setting cratesRequired to 2 will break existing installations that have units deployed with only one crate.

Reply:
In my opinion, that's just a parameter with a default value, that can be changed if the user does not want the default.
Or do you mean something else?

Reply:
When I ran it with my saved PGAW state files, the Patriots did not spawn because a crate was missing.  Anyone using persistent spawns that doesn't change it back will have a problem.